env.js

```diff
@@ -219,8 +219,6 @@ if (shouldValidateEnv) {
       );
     }
 
-    console.error(...messages);
```

Comment:
Oops! This one is intended.

Reply:
RIght!!! I will disable the rule for console.error 😓 Thanks